4.6/Dockerfile

```diff
@@ -34,4 +34,10 @@ RUN apt-get update \
 	&& make install-strip \
 	&& cd .. \
 	&& rm -rf "$dir" \
-	&& apt-get purge -y --auto-remove curl gcc g++ wget
+	&& apt-get purge -y --auto-remove curl wget
+
+# ensure that alternatives are pointing to the new compiler and that old one is no longer used
+RUN update-alternatives --remove cc /usr/bin/gcc \
+	&& unlink /usr/bin/gcc \
+	&& unlink /usr/bin/g++ \
+    && update-alternatives --install /usr/bin/cc cc /usr/local/bin/gcc 100
```

Comment:
Whitespace here is off (tabs vs spaces), including Git mentioning `No newline at end of file`.

Also, shouldn't we use a higher priority than 100 (thinking like 999 or something similarly high) to make sure that the Debian packages are less likely to cause trouble in the future?